hassio/src/dialogs/network/dialog-hassio-network.ts

```diff
@@ -158,7 +164,7 @@ export class DialogHassioNetwork extends LitElement implements HassDialog {
                 class="flex-auto"
                 id="nameservers"
                 label="DNS servers"
-                .value="${this._device!.data.nameservers.join(", ")}"
+                .value="${this._device!.data.nameservers.join(",")}"
```

Comment:
Why not save this as string instead of keep joining and splitting it on every change and split when we save